scientific_library/tvb/simulator/monitors.py

```diff
@@ -371,8 +368,7 @@ class Projection(Monitor):
 
     region_mapping = Attr(
         RegionMapping,
-        required=False,
```

Comment:
The way currently `Projection.config_for_sim` is written, `Projection. region_mapping ` attribute can be missing in case of a surface based simulation, and then the RM is read from there.

Reply:
Done.